vms/authentication/tests/test_services.py

```diff
@@ -1,3 +1,14 @@
 from django.test import TestCase
+from authentication.views import login_process
+from django.test.client import RequestFactory
+
 
 # Create your tests here.
+class TestLoginProcess(TestCase):
+
+	def testLoginViewRateLimit(self):
```

Comment:
@rituranjna Can you increase the number to 35 or higher? We want to verify something.